packages/instrumentation/src/ActiveTracingHelper.ts

```diff
@@ -85,6 +88,11 @@ export class ActiveTracingHelper implements TracingHelper {
     const tracer = this.tracerProvider.getTracer('prisma')
     const context = options.context ?? this.getActiveContext()
     const name = `prisma:client:${options.name}`
+
+    if (shouldIgnoreSpan(name, this.ignoreSpanTypes)) {
+      return callback()
+    }
```

Comment:
What happens if we are in a parent span created by user in the application code, and `prisma:client:operation` is skipped? Won't this lead to the child spans being created and linked to the application span anyway?

Reply:
Based on the functional tests child spans are fully filtered out if their parent is filtered out. Why do you think this might not be the case? 🤔

Reply:
Updated the test as we discussed